packages/teams-js/src/public/interfaces.ts

```diff
@@ -544,14 +547,17 @@ export interface DialogSize {
   width: DialogDimension | number;
 }
 
+/**
+ * Data structure to describe dialog information needed to open open a url based dialog.
```

Comment:
I added a summary line, but not sure how to create a link to the actual function without importing the function to this file. If we think, linking will help users, I will figure a way.

Reply:
Also was not sure how to write a summary for dialogInfo interface below

Reply:
I do think you have to import the function in order to link to it. If dialog is already linking to this then maybe it's okay to not link in both directions (especially if this structure might be used in more than one capability or function).

Reply:
I think, in dioalog.ts file, it is straight forward to find the dialogDimension interfaces. So, I did not link them. It makes my parameter summary a little smooth. 

In dialog.ts, do we prefer below, or do we want to leave out the `of type {@linkcode UrlDialogInfo}` part out? 
   * @param urlDialogInfo - An object of type {@linkcode UrlDialogInfo} containing the parameters of the dialog module.
   * @param submitHandler - Handler of type {@linkcode DialogSubmitHandler} that triggers when a dialog calls the {@linkcode submit} function or when the user closes the dialog.